app/controllers/leads_controller.rb

```diff
@@ -47,7 +47,7 @@ def edit
                   :to   => @lead.phone,
                   :from => ENV['TWILIO_PHONE_NUMBER']
     })
-    @messages = (messages_from_lead + messages_from_call_converter)
+    @messages = (messages_from_lead + messages_from_call_converter).sort {|m,o| o.date_sent.to_d <=> m.date_sent.to_d}
```

Comment:
`to_d` converts the date which is currently stored as a string to a date so that the .sort can sort it properly.